### ISSUE TITLE
Sticky ticket Qty.

### ISSUE THREAD

**Issue by tarkal** [ff36/jnoc#14]
USER: matthew.blumenthal@solid.com 
MESSAGE: It would be really cool if the number of tickets displayed could be sticky when it is changed. On a desktop, I tend to like more than 24 displayed at a time. I'm constantly having to change it when I refresh the page for any reason. 
DEVICE: DeviceCategory [category=PERSONAL_COMPUTER, icon=desktop.png, infoUrl=/list-of-ua/device-detail?device=Personal computer, name=Personal computer] 
FAMILY: CHROMIUM 
OS: OperatingSystem [family=LINUX, familyName=Linux, icon=linux_ubuntu.png, name=Linux (Ubuntu), producer=Canonical Ltd., producerUrl=http://www.canonical.com/, url=http://www.ubuntu.com/, versionNumber=VersionNumber [groups=[, , ], extension=]] 
BROWSER: BROWSER 
BROWSER VERSION: VersionNumber [groups=[39, 0, 2171, 65], extension=]

**Comment by cantorfan** [comment]
I don't understand exactly. currently, on the bottom of ticket list page, there is a drop down list to allow user select how many tickets per page.  What's we need to do?

**Comment by tarkal** [comment]
Yes but currently when the page loads it will always load the minimum number of tickets per page (call it 50). If you change this to 100 for example and reload the page it will go back to 50. When a user changes the value to say 100 it should place a cookie/local storage token to remember the value when the user reloads the page.

N.B. I think it currently does this by calculating the total number available and performing some division function. These values should be set. ( maybe something like 25, 50, 75, 100, 150, 200, 500).
